lib/covid19_orientation_web/schemas/conclusion.ex

```diff
@@ -14,14 +14,14 @@ defmodule Covid19OrientationWeb.Schemas.Conclusion do
     properties: %{
       code: %Schema{
         type: :string,
-        enum: ["FIN1", "FIN2", "FIN3", "FIN4", "FIN5", "FIN6", "FIN7", "FIN8", "FIN9"],
+        enum: ["orientation_SAMU", "FIN2", "FIN3", "FIN4", "FIN5", "FIN6", "FIN7", "FIN8", "FIN9"],
```

Comment:
Do you have the reference ?

Reply:
Column B72 of the reference spreadsheet: "orientation" => "FIN1". This is the only orientation taken into account in the spreadsheet, I read this as "other orientations don't need to be stored", which makes sense, as long as all data are collected. WDYT?

Reply:
I think you got C73? I'd rather store them, also is simpler.

Reply:
« Exceptions are always the source of more exceptions »

Reply:
Mhh.. no, I got B72, see below.

![B72](https://user-images.githubusercontent.com/63500/78186285-c6463000-746c-11ea-9e80-fef95c48eb31.png)

Reply:
Agreed about the exceptions... except that in this case, the `orientation_SAMU` variable name seems (deadly) critical!